monitoring/base/prometheus/alert_rules/ingress-watcher.yaml

```
@@ -0,0 +1,21 @@
+groups:
+  - name: ingress-watcher
+    rules:
+      - alert: IngressDown
+        expr: |
+          rate(ingresswatcher_http_get_successful_total{code="200 OK"}[5m]) / on(job,path) rate(ingresswatcher_http_get_total[5m]) < 0.95
```

I think it is better to use `ignoring`. If using `on`, this rule will break when new target labels are added.
```suggestion
          rate(ingresswatcher_http_get_successful_total{code="200 OK"}[5m]) / ignoring(code) rate(ingresswatcher_http_get_total[5m]) < 0.95
```